zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupManager.java

```diff
@@ -606,9 +618,25 @@ public synchronized void initialize() throws IOException {
       } catch (Exception e) {
         throw new BackupException(e.getMessage(), e);
       }
+
+      // create metric-related classes for timetable backup
+      TimetableBackupStats timetableBackupStats = new TimetableBackupStats();
+      timetableBackupBean = new TimetableBackupBean(timetableBackupStats, serverId);
+      try {
+        MBeanRegistry.getInstance().register(timetableBackupBean, null);
```

Comment:
Shall we register TimetableBackupBean as child bean of BackupBean, since it's a subprocess of the entire backup process?